db/postgres-init.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I would rename this file to posgres-dev-init.sh since it’s gonna be used only for development env